manoseimas/settings/base.py

```diff
@@ -157,6 +158,7 @@
     'manoseimas.compat',            # depends on: solutions
     'manoseimas.widget',
     'manoseimas.lobbyists',
+    'manoseimas.compatibility_test',
```

Comment:
compatibility_test will be main application I'm working on! It's new version of compat ;) (Suderinamumo su seimo nariais/partijomis testas)